test/servico/Cst22Test.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+
+import datamapper.PopulateDB;
+
+import dominio.Usuario;
+
+import java.util.List;
+
+import modelo.UsuarioModel;
+
+import org.junit.BeforeClass;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+/**
+ *
+ * @author Victor Messina
+ */
+public class Cst22Test {
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+
+    @Test

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def modifier' have incorrect indentation level 4, expected level should be 2.](https://www.codacy.com/app/calebepb/substituicao/file/2777960168/issues/source?bid=3364462&fileBranchId=3364479#l36)